Woosmap/geofencing-core-android-sdk
# Add source header in the API request

# Description
## Issue
closes #6
closes https://github.com/Woosmap/woosmap/issues/1063

## What
Adding origin/source in the header of Search API and Distance API request

## How
1. Created a common method to create a request for all the APIs
2. Added following headers in the created request
```
"X-Api-Key": _key,
"X-Android-Identifier": packageInfo.packageName,
"X-SDK-Source": @"geofence-sdk",
"X-AK-SDK-Platform": @"Android",
"X-AK-SDK-Version": [self getVersion]
```

## Related PRs
None

# Testing
## Automated tests
- [x] Manual Tests cover the change  


## Steps to test or reproduce
1. Run the sample app in debug mode (Emulator or device)
2. Check API calls in Network Inspector tab.
3. Check if all the API calls have above request headers present in them.

# Deployment
## Strategy

## Migrations
No.
